### PR TITLE
Cross tenant fields for audit logs

### PR DESCRIPTION
**Pull request by besiler**
Added homeTenantId and homeTenantName to directory audit. We missed this when pushing the docs for https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5102844?_a=files&path=/Changelog/5102844-SignIn-and-Audit-Updates-for-Q1-in-PROD-MC-FF/Microsoft.AAD.Reporting.json

Somehow, the changelog generated by that PR didn't pick up the changes to directoryAudit. I made up the changelog entries in this PR so please take a look at them to make sure they look OK

### CODE REVIEW

File: api-reference/beta/resources/directoryaudit.md [hunk 1]
```diff
@@ -24,13 +24,15 @@ Represents the directory audit items and its collection.
...
 | correlationId       | Guid                                                | Indicates a unique ID that helps correlate activities that span across various services. Can be used to trace logs across services.                                                                                                                                    |
+|homeTenantId|Guid|The unique identifier of the tenant that the user who initiated the auditable activity belongs to. This value is often populated in situations when an organization has granted permission to an outside organization (such as a CSP or Microsoft support) to make changes to their tenant settings.|
+|homeTenantName|String|The display name of the tenant that the user who initiated the auditable activity belongs to. This value is often populated in situations when an organization has granted permission to an outside organization (such as a CSP or Microsoft support) to make changes to their tenant settings.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/directoryaudit.md:35]: Hi @besiler - This is incorrect. The two properties were added to the **auditUserIdentity** complex type, _not_ to **directoryAudit** resource type - and you already documented them. See https://docs.microsoft.com/en-us/graph/api/resources/auditUserIdentity?view=graph-rest-beta

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @besiler - The two properties were added to the https://docs.microsoft.com/en-us/graph/api/resources/auditUserIdentity?view=graph-rest-beta complex type and they are already in the public docs. I recommend undoing the changes in directoryaudit.md and the changelog and keeping the updated examples instead.

**Comment by besiler** [comment]
@FaithOmbongi thank you so much for keeping me honest that the user properties were already added in https://docs.microsoft.com/en-us/graph/api/resources/auditUserIdentity?view=graph-rest-beta! I couldn't find those properties earlier to show the cross tenant team, so I thought I had missed it. 

I'll follow your suggestions above to keep the example but drop the new properties

File: api-reference/beta/resources/directoryaudit.md (hunk 1 above)
Comment (besiler) [reply api-reference/beta/resources/directoryaudit.md]: Removed the field descriptions from directoryaudit.md and removed the changelog entry as requested by Faith

**Pull request merged**